java/connector-node/risingwave-sink-iceberg/src/main/java/com/risingwave/connector/IcebergSinkUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2023 RisingWave Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.risingwave.connector;
+
+import static io.grpc.Status.INVALID_ARGUMENT;
+import static io.grpc.Status.UNIMPLEMENTED;
+
+import com.risingwave.connector.api.TableSchema;
+import com.risingwave.proto.Data;
+import io.grpc.Status;
+import java.util.List;
+import java.util.Map;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
+
+public class IcebergSinkUtil {
+    private static Type convertType(Data.DataType.TypeName typeName) {
+        switch (typeName) {
+            case INT16:
```

Comment:
For int16, iceberg may have things like `Types.ShortType.get()`?

Reply:
I didn't find a type like ‘ ShortType’ in ‘Types’

Reply:
I didn't find a type like `ShortType` in iceberg Types